CREDITS.md

```diff
@@ -3,3 +3,4 @@
 <div>Icons made by <a href="http://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
 <div>Icons made by <a href="https://www.flaticon.com/authors/dave-gandy" title="Dave Gandy">Dave Gandy</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
 <div>Icons made by <a href="https://www.flaticon.com/authors/vectors-market" title="Vectors Market">Vectors Market</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+<div><a href="https://dribbble.com/shots/2316666-Profile-card-UI" title="UI">The UI</a> has been inspired by <a href="https://dribbble.com/sokratus" title="dribble">Sokratus</a> from <a href="https://dribble.com/" title="dribble">Dribble</a>.</div>
```

Comment:
Lets change it to something like this

`<div>The UI has been inspired by <a href="https://dribbble.com/sokratus" title="dribble">Sokratus's</a> <a href="https://dribbble.com/shots/2316666-Profile-card-UI" title="UI">work</a> from <a href="https://dribble.com/" title="dribble">Dribble</a>.</div>`

How does this looks? Any thoughts about this?